martypy/ClientMV2.py

```diff
@@ -407,6 +408,95 @@ def is_conn_ready(self) -> bool:
             return False
         return self._initComplete and (self.lastSubscribedMsgTime is not None)
 
+    def _valid_addon(self, add_on: str) -> bool:
```

Comment:
```suggestion
    def _is_valid_disco_addon(self, add_on: str) -> bool:
```
This only checks disco addons, not other addons, right?